test/unit/vy_mem.c

```diff
@@ -297,12 +297,280 @@ test_iterator_restore_after_insertion(void)
 	footer();
 }
 
+static const char *
+lsn_str(int64_t lsn)
+{
+	char *buf = tt_static_buf();
+	if (lsn == INT64_MAX) {
+		strlcpy(buf, "INT64_MAX", TT_STATIC_BUF_LEN);
```

Comment:
Nit: you could just return "INT64_MAX" here.

Reply:
Right :) Fixed here and in `test/unit/vy_cache.c` as well.